docs/download.md

```diff
@@ -85,27 +85,24 @@ CUDA 11.8 through CUDA 12.0.
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
 * Verify the signature for Scala 2.12 jar:
-    `gpg --verify rapids-4-spark_2.12-24.12.1.jar.asc rapids-4-spark_2.12-24.12.1.jar`
+    `gpg --verify rapids-4-spark_2.12-25.02.0.jar.asc rapids-4-spark_2.12-25.02.0.jar`
 * Verify the signature for Scala 2.13 jar:
-    `gpg --verify rapids-4-spark_2.13-24.12.1.jar.asc rapids-4-spark_2.13-24.12.1.jar`
+    `gpg --verify rapids-4-spark_2.13-25.02.0.jar.asc rapids-4-spark_2.13-25.02.0.jar`
 
 The output of signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
-* Add repartition-based algorithm fallback in hash aggregate  
-* Support Spark function months_between
-* Support asynchronous writing for Parquet files
-* Add retry support to improve sub hash-join stability
-* Improve JSON scan and from_json
-* Improved performance for CASE WHEN statements comparing a string column against multiple values
-* Falling back to the CPU for ORC boolean writes by the GPU due to a bug in cudf's ORC writer
-* Fix a device memory leak in timestamp operator in `incompatibleDateFormats` case
-* Fix a host memory leak in GpuBroadcastNestedLoopJoinExecBase when `spillableBuiltBatch` is 0
+* Support Spark function Bin
+* Improve spark metrics: Print the batch size information to executor log
+* Refine filter push down to avoid double evaluation
+* Grab the GPU Semaphore when reading cached batch data with the GPU to avoid a GPU OOM case
+* Add an option to disable measuring buffer copy to improve large shuffle large partition serialization
+* Support group-limit optimization for ROW_NUMBER
```

Comment:
```suggestion
* Support the Spark functions Bin and TruncDate
* Support group-limit optimization for ROW_NUMBER
* Improve Spark metrics: Print the batch size information to executor log
* Refine filter push down to avoid double evaluation
* Grab the GPU Semaphore when reading cached batch data with the GPU to avoid a GPU OOM case
* Add an option to disable measuring buffer copy to improve large shuffle large partition serialization
```